superset/db_engine_specs/presto.py

```diff
@@ -20,17 +20,20 @@
 import logging
 import re
 import time
+import jwt
+import os
+import requests
 from abc import ABCMeta
 from collections import defaultdict, deque
 from datetime import datetime
 from re import Pattern
 from textwrap import dedent
-from typing import Any, cast, Optional, TYPE_CHECKING
+from typing import Any, cast, Optional, TYPE_CHECKING, Dict, List
 from urllib import parse
 
 import pandas as pd
 import simplejson as json
-from flask import current_app
+from flask import current_app, request
```

Comment:
I would highly suggest to avoid importing `request` and `requests` in the same scope - they are to similar - easy to be confused.

Reply:
Done

Reply:
Cool, please close this conversation, because I don't have permission to do this (there is no button).